### DISCUSSION TITLE
Running local postgres db with supabase start fails due to no storage to persist session.

### DISCUSSION THREAD

**Discussion by qwerqy** [supabase/supabase#15886] [Questions]
Hello! I recently started using Supabase on a NextJS 13+ with app router and i am using the auth helper for nextjs.  I am trying to setup supabase for my local env via supabase init. Upon running supabase start, i get an error saying there are no storage to persist session. 

After checking the source code for the auth helper, i see that persistSession defaults to true and the storage is stored in the browser. I am pretty sure this was the cause of the problem but it makes sense since i am not running my supabase client inside of a server. 

My question is, if i were to develop this project without a server, is it best just to not worry about migration files and just make adjustments on the dashboard? My project is fairly new and yet to be scaled.

**Comment by silentworks** [comment]
It seems you are referring to two different things here. `supabase start` itself wouldn't give you an error of `no storage to persist session`, this would come from your application itself when you start it. This shouldn't happen with the auth-helpers as its using cookies as storage so there might be something else happening in your code.

If you are developing only on the client-side you don't need to use the auth-helpers at all, you can just use the regular `@supabase/supabase-js` in that case.

**Comment by qwerqy** [comment]
I see. Thank you. I will check it out 😁

On Wed, 19 Jul 2023 at 5:59 PM, Andrew Smith ***@***.***>
wrote:

> It seems you are referring to two different things here. supabase start
> itself wouldn't give you an error of no storage to persist session, this
> would come from your application itself when you start it. This shouldn't
> happen with the auth-helpers as its using cookies as storage so there might
> be something else happening in your code.
>
> If you are developing only on the client-side you don't need to use the
> auth-helpers at all, you can just use the regular @supabase/supabase-js
> in that case.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/supabase/discussions/15886#discussioncomment-6487574>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMIV5FJIJ2JLNIZL2TX2RDXQ64ZPANCNFSM6AAAAAA2PQH6F4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Thank you,
Amin Roslan
aminroslan.com
